### PR TITLE
added new contributors section

### PR DESCRIPTION
**Pull request by anshitmishra**
**Notes for Reviewers**

This PR fixes #

Describe the changes you've made:
added new contributors section

Screenshots for the change:
Screenshot (53)
Screenshot (52)

### CODE REVIEW

**Review by sudhanshutech — COMMENTED** [review]
We don't need the bot as contributor, so can you remove it? Rest all looks good
Is this is generated dynamically?

**Comment by anshitmishra** [comment]
> We don't need the bot as contributor, so can you remove it? Rest all looks good Is this is generated dynamically?

yes everything is dynamic

**Comment by anshitmishra** [comment]
Screenshot (56)
added new changes

**Comment by Yashsharma1911** [comment]
this looks great @anshitmishra just wondering why you didn't add title in contributor section as given in Figma do you have any reason for that

**Comment by anshitmishra** [comment]
> this looks great @anshitmishra just wondering why you didn't add title in contributor section as given in Figma do you have any reason for that

I believe mere code push karni kii baad design main kuch changes aai hain...

soon I will update with new design @Yashsharma1911

**Comment by ArunPatel02** [comment]
@anshitmishra can we merge this or you are working on this pr ?

**Comment by anshitmishra** [comment]
> @anshitmishra can we merge this or you are working on this pr ?

yes i believe everything is done by me if is there anything left let me know otherwise we can merge.

**Review by Yashsharma1911 — APPROVED** [review]

File: src/Components/Alternative/contributors/Contributors.module.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    .mainHeading {
+        width: 90%;
+    }
+
+    .mainHolder {
+        width: 90%;
+    }
+}
+
+/* Large devices (laptops/desktops, 992px and up) */
```
Comment (Yashsharma1911) [inline src/Components/Alternative/contributors/Contributors.module.css:151]: thanks for adding comments

File: src/Components/Alternative/contributors/contributors.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import styles from "./Contributors.module.css"
+import { useEffect } from "react";
+export default function Contributers() {
+    const [data, setData] = useState([]);
+    const FetchData = async () => {
+        let fetchList = await fetch("https://api.github.com/repos/GDSCITM/GDSCITM-official-website/contributors?anon=1")
+        let data = await fetchList.json()
+        console.log(data, "asdasdas");
```
Comment (Yashsharma1911) [inline src/Components/Alternative/contributors/contributors.js:9]:
```suggestion
```

**Review by Yashsharma1911 — DISMISSED** [review]

**Pull request merged**